azurion/properties/Requirement6.mcf

```diff
@@ -0,0 +1 @@
+exists dirJ:JoystickDirection.[true*.moveTableJoystick(dirJ)]mu X. ([!moveTable(translateJoystickToTableMovement(dirJ))]X && <true>true)
```

Comment:
It should be ``forall`` instead of ``exists`` here I think. Because we want that for every joystick direction ``moveTable`` to happen.
Also, here we should add the condition that ``operationOngoing`` and ``emergencyMode`` are false so that the ``moveTable`` can happen.

Reply:
As far as I understand from examples `exists` is used for any value so I thought this modal formula means: for any joystick direction dirJ moveTableJoystick is moving the table....